third_party/openenclave/Makefile

```diff
@@ -5,7 +5,7 @@ TOP=$(abspath $(CURDIR)/../..)
 BUILDDIR=${TOP}/build
 
 URL=https://github.com/openenclave/openenclave
-BRANCH=mystikos.v6
+BRANCH=master
```

Comment:
It is important to lock this down to a specific commit. Otherwise, it is not easily possible to checkout an older version of Mystikos and build it - we won't know what commit of OE a specific commit of Mystikos worked with.